charts/azuredisk-csi-driver/templates/azuredisk-csi-driver.yaml

```diff
@@ -0,0 +1,141 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-azuredisk
+  namespace: {{ .Release.Namespace }}
+{{ include "adcd.labels" . | indent 2 }}
+    app: csi-azuredisk
+spec:
+  selector:
+    matchLabels:
+      app: csi-azuredisk
+  template:
+    metadata:
+{{ include "adcd.labels" . | indent 6 }}
```

Comment:
what dose `adcd` mean?

Reply:
Emmm...
`adcd = azure disk csi driver`, maybe it's not intuitive.

Reply:
maybe `azuredisk` is better